include/libtorrent/aux_/vector.hpp

```diff
@@ -143,6 +143,20 @@ namespace libtorrent { namespace aux {
 		return {vec.data() + static_cast<type>(begin), vec.data() + static_cast<type>(end)};
 	}
 
+	// TODO: find a better place for this function
```

Comment:
interesting, didn't know about `boost::numeric_cast`, do you want to do something else about it? any other name suggestion?